src/components/SentList/index.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react"
+import SentItem from "../SentItem"
+
+function SentList() {
+  const [emails, setEmails] = useState([])
+  function getEmailData() {
+    fetch("https://email-client-api.dev.io-academy.uk/emails/sent")
+      .then((res) => res.json())
+      .then((emailData) => {
```

Comment:
To be consistent with your other fetch requests, you should remove the () around res and emailData